queue/python/queue.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+Queue = {'front': None, 'back': None}
+
+class Node:
+  def __init__(self, data, next):
+    self.data = data
+    self.next = next  
+    
+def Enqueue(Queue, element):
+  N = Node(element, None)
+  if Queue['back'] == None:
+    Queue['front'] = N
+    Queue['back'] = N
+  else:
+    Queue['back'].next = N
+    Queue['back'] = Queue['back'].next
+    
+def Dequeue(Queue):
+  if Queue['front'] != None:
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp754ta467/queue/python/queue.py
+++ b/tmp/tmp754ta467/queue/python/queue.py
@@ -35,4 +35,4 @@
   print (Dequeue(Queue))
 
 if __name__=="__main__":
-  main()+  main()
```